sky/clouds/gcp.py

```diff
@@ -328,13 +348,32 @@ def check_credentials(self) -> Tuple[bool, Optional[str]]:
         return True, None
 
     def get_credential_file_mounts(self) -> Dict[str, str]:
+        # Create a backup of the config_default file, as the original file can
```

Comment:
Nit: say this backup file will be uploaded to the remote clusters and applied appropriately.

Reply:
Added. Thanks!